### PR TITLE
Update the schema to https instead of http URLs

### PR DESCRIPTION
**Pull request by TobiasNx**
Not only the transformation but also the schema needs to be updated.

https://github.com/hbz/lobid-resources/issues/1343

### CODE REVIEW

**Comment by TobiasNx** [comment]
needed to add more changes to the transformation and the schema to get all `http` @dr0i could you have a look at the transformation why it fails? also @acka47 could you have a look at the schema changes?

File: src/test/resources/alma/(CKB)5280000000199164.json [hunk 1]
```diff
@@ -85,8 +85,8 @@
       "label" : "Evangelisch-Lutherisches Missionswerk Leipzig"
     },
     "role" : {
-      "id" : "http://id.loc.gov/vocabulary/relators/isb",
-      "label" : "Herausgeber/in"
+      "id" : "https://id.loc.gov/vocabulary/relators/isb",
+      "label" : "isb"
```
Comment (dr0i) [inline src/test/resources/alma/(CKB)5280000000199164.json:89]: Why not keep the more verbose label?

File: src/test/resources/alma/(DE-605)HT000161712.json [hunk 2]
```diff
@@ -166,8 +166,8 @@
       "id" : "https://d-nb.info/gnd/1023590484"
     },
     "role" : {
-      "id" : "http://id.loc.gov/vocabulary/relators/edt",
-      "label" : "Herausgeber/in"
+      "id" : "https://id.loc.gov/vocabulary/relators/edt",
+      "label" : "edt"
     }
```
Comment (dr0i) [inline src/test/resources/alma/(DE-605)HT000161712.json:171]: dito verbose

File: src/test/resources/alma/(DE-605)HT004285445.json [hunk 3]
```diff
@@ -45,8 +45,8 @@
       "id" : "https://d-nb.info/gnd/134465202"
     },
     "role" : {
-      "id" : "http://id.loc.gov/vocabulary/relators/cmp",
-      "label" : "Komposition"
+      "id" : "https://id.loc.gov/vocabulary/relators/cmp",
+      "label" : "cmp"
     }
```
Comment (dr0i) [inline src/test/resources/alma/(DE-605)HT004285445.json:50]: better verbose

File: src/test/resources/alma/(DE-605)HT005207972.json [hunk 4]
```diff
@@ -114,8 +114,8 @@
       "type" : [ "Person" ]
     },
     "role" : {
-      "id" : "http://id.loc.gov/vocabulary/relators/aut",
-      "label" : "Autor/in"
+      "id" : "https://id.loc.gov/vocabulary/relators/aut",
+      "label" : "aut"
     }
```
Comment (dr0i) [inline src/test/resources/alma/(DE-605)HT005207972.json:119]: dito

**Review by dr0i — CHANGES_REQUESTED** [review]
See inline comments re verbosity. Haven't marked them all.

**Comment by TobiasNx** [comment]
> See inline comments re verbosity. Haven't marked them all.

This seems to be due to the labelmaker not knowing about the changes from http -> https.

**Comment by dr0i** [comment]
@UBmakla or @hagbeck : is the change of `http` to `https` in some way affecting you, or doesn't it matter?

**Comment by acka47** [comment]
We should be careful about this. For DNB URIs and probably also for ZDB URIs this is fine as they switched to HTTPS URIs but MARC relators still use HTTP-only URIs (and do not provide a `sameAs` link or similar to the https URI), see e.g. https://id.loc.gov/vocabulary/relators/ill.html. So we should stick with that as this is important for the RDF representation.

**Review by acka47 — CHANGES_REQUESTED** [review]
As noted in the other comment, we should continue using http-only URIs for MARC relator codes.

**Comment by hagbeck** [comment]
The change to `https` is ok as long as the resulting URLs are not broken then ;-) 
We are using parts of some URL fields for selections and need to update these rules in some cases. But this is ok.

**Comment by hagbeck** [comment]
Btw.: URLs to table of contents provided by hbz are currently `http`-only, too.

**Comment by dr0i** [comment]
Conclusion so far:
Be aware of the generic URIs - even if e.g. `curl -L  --header "accept: text/plain" 'https://id.loc.gov/vocabulary/relators/aut'` brings up the proper `ntriples`, it's not reflected in the underlying data as [@acka47 noted ](https://github.com/hbz/lobid-resources/pull/1346#issuecomment-1152368724) as you can see in the data and in the missing `sameAs`. That's also true for our own [RPB-Skos](http://purl.org/lobid/rpb).
After you have checked `https` is ok, I would recommend to add this `https` variants to the shared label files (rather than have a copy of it ) - because it makes things easier to handle and consume and easier to exchange the label files).

**Pull request closed without merging**